freqtrade/exchange/binance.py

```diff
@@ -136,17 +135,19 @@ def fill_leverage_brackets(self):
                 else:
                     leverage_brackets = self._api.load_leverage_brackets()
 
-                for pair, brackets in leverage_brackets.items():
-                    self._leverage_brackets[pair] = [
-                        [
-                            min_amount,
-                            float(margin_req)
-                        ] for [
-                            min_amount,
-                            margin_req
-                        ] in brackets
-                    ]
-
+                for pair, brkts in leverage_brackets.items():
+                    [amt, old_ratio] = [None, None]
+                    brackets = []
+                    for [notional_floor, mm_ratio] in brkts:
+                        amt = ((float(notional_floor) * (mm_ratio - old_ratio)) +
+                               amt) if old_ratio else 0
+                        old_ratio = mm_ratio
+                        brackets.append([
+                            float(notional_floor),
+                            mm_ratio,
+                            amt,
+                        ])
+                    self._leverage_brackets[pair] = brackets
```

Comment:
@xmatthias It's something in this for loop that's causing all the `freqtrade does not support isolated futures on binance` errors. If I replace this code with what was there before I don't get the errors. I've tried taking out lines like `amt = ` ... but still get the errors